kernel/src/arch/x86_64/process.rs

```diff
@@ -102,93 +94,135 @@ lazy_static! {
                 };
                 process_logs.push(local_log_arc);
             }
-            return process_logs;
+            return process_logs;*/
+            unimplemented!("Need to get NodeReplicated from controller")
         }
 
-        let process_logs = {
-            let mut process_logs = Box::try_new(ArrayVec::new()).expect("Can't initialize process log vector.");
-            for _pid in 0..MAX_PROCESSES {
-                let log = Arc::try_new(
-                    Log::<<NrProcess<Ring3Process> as Dispatch>::WriteOperation>::new(LARGE_PAGE_SIZE),
-                )
-                .expect("Can't initialize process logs, out of memory.");
-                process_logs.push(log);
-            }
-            process_logs
-        };
+        // We want to allocate the logs in controller shared memory
```

Comment:
I think comment is outdated. We are allocating below from local memory below (local_shmem_affinity())